lib/tarantool_server.py

```diff
@@ -464,8 +465,9 @@ def seek_wait(self, msg, proc=None, name=None):
                     f.seek(cur_pos, os.SEEK_SET)
                     continue
                 if re.findall(msg, log_str):
-                    return
+                    return True
                 cur_pos = f.tell()
+        return False
```

Comment:
The timeout situation does not look much different from just startup error. Can we raise `TarantoolStartError` with a descriptive error message in the case (it seems, we should add a parameter into its constructor and `__str__` method)?

Reply:
Do you want to raise the exception here? I'm not confident due to this function is about seeking a pattern. Of course, for a proc, which does not exist, is important. But in case of a pattern absence, I suggest returning the bool value. I may add the raise condition wait_until_started.

Reply:
Okay.